### PR TITLE
disable unused targets

### PR DESCRIPTION
**Pull request by AndreyMlashkin**

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -77,8 +77,8 @@ endmacro()
 set(COMMON_TEST_FILES
     "manifest"
     "detail/huang_lu"
-    "gate_id"
-    "utils/connectedness_check"
+    #"gate_id"
```
Comment (Iluvmagick) [inline test/CMakeLists.txt:80]: Can we re-enable these two tests, or do they fail?

**Review by Iluvmagick — APPROVED** [review]

File: run_tests.sh [hunk 2]
```diff
@@ -1,77 +1,7 @@
 #!/usr/bin/env bash
```
Comment (knazarov) [inline run_tests.sh:1]: I don't think you need this file now, as running tests is only one command. But that's not a big deal anyway. Keep it if you want.

**Review by knazarov — APPROVED** [review]

File: test/CMakeLists.txt (hunk 1 above)
Comment (AndreyMlashkin) [reply test/CMakeLists.txt]: They are actually working. I don't know, why they were not in the original list of the tests to run...

File: run_tests.sh (hunk 2 above)
Comment (AndreyMlashkin) [reply run_tests.sh:1]: I am merging the whole blueprint to crypto3 after and it will go anyway

**Pull request merged**